integration/spark2/src/main/commonTo2.2And2.3/org/apache/spark/sql/hive/CarbonAnalyzer.scala

```diff
@@ -42,10 +42,10 @@ class CarbonAnalyzer(catalog: SessionCatalog,
     var logicalPlan = analyzer.execute(plan)
     logicalPlan = CarbonPreAggregateDataLoadingRules(sparkSession).apply(logicalPlan)
     logicalPlan = CarbonPreAggregateQueryRules(sparkSession).apply(logicalPlan)
-    if (mvPlan != null) {
+    if (mvPlan != null && !logicalPlan.isInstanceOf[Command]) {
```

Comment:
does command also need CarbonPreAggregateQueryRules?

Reply:
both CarbonPreAggregateDataLoadingRules and CarbonPreAggregateQueryRules are no need for Command. I will fix it